src/braket/circuits/translations.py

```diff
@@ -122,44 +129,44 @@ def _braket_result_to_result_type(result: Results) -> None:
     raise TypeError(f"Result type {type(result).__name__} is not supported")
 
 
-def braket_result_to_result_type(result: Results) -> None:
+def braket_result_to_result_type(result: Results) -> None:  # noqa: D103
     return _braket_result_to_result_type(result)
 
 
 @_braket_result_to_result_type.register(Amplitude)
-def _(result):
+def _(result: Results) -> Amplitude:
```

Comment:
Will we be able to configure ruff to not worry about these for functions that won't/can't actually be called explicitly?

Reply:
We can using `noqa` however I think in some cases, it does help for readability.